plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcClient.java

```diff
@@ -142,9 +145,6 @@ default void setColumnComment(ConnectorSession session, JdbcTableHandle handle,
 
     String buildInsertSql(JdbcOutputTableHandle handle, List<WriteFunction> columnWriters);
 
-    Connection getConnection(ConnectorSession session, JdbcOutputTableHandle handle)
```

Comment:
In some sense, it was group per feature (the method above this one is for insert/ctas, and this is too), not by performed operation (both getConnection together), but i am fine either way